### PR TITLE
[MNOE-1028] Update subscription events upon updating subscriptions

### PR DESCRIPTION
**Pull request by iseessel**

### CODE REVIEW

File: src/app/views/orders/order.controller.coffee [hunk 1]
```diff
@@ -94,7 +94,11 @@
             toastr.success('mnoe_admin_panel.dashboard.subscriptions.modal.approve_subscriptions.toastr_success', {extraData: {subscription_name: vm.order.product.name}})
           ->
             toastr.error('mnoe_admin_panel.dashboard.subscriptions.modal.approve_subscriptions.toastr_error', {extraData: {subscription_name: vm.order.product.name}})
-        )
+        ).then(() ->
+            MnoeProvisioning.getSubscriptionEvents(vm.orderId, vm.orgId)
```
Comment (adamaziz15) [inline src/app/views/orders/order.controller.coffee]: Can we refactor this into a function like `refreshSubscriptionEvents` maybe? And then just pass that function to both `.then`s in this controller.

Comment (iseessel) [reply src/app/views/orders/order.controller.coffee]: Ya probably better

File: src/app/views/orders/order.controller.coffee [hunk 2]
```diff
@@ -132,7 +135,7 @@
             toastr.success('mnoe_admin_panel.dashboard.subscriptions.widget.list.toastr_success', {extraData: {subscription_name: vm.order.product.name}})
           ->
             toastr.error('mnoe_admin_panel.dashboard.subscriptions.widget.list.toastr_error', {extraData: {subscription_name: vm.order.product.name}})
-        )
+        ).finally(() -> getSubscription())
```
Comment (adamaziz15) [inline src/app/views/orders/order.controller.coffee]: Where is this `getSubscription` function defined?

Comment (iseessel) [reply src/app/views/orders/order.controller.coffee]: @adamaziz15 My b.

**Pull request merged**